tools/mmp/driver.cs

```diff
@@ -1347,13 +1347,17 @@ static int Compile ()
 				args.Append ("-o ").Append (StringUtils.Quote (AppPath)).Append (' ');
 				args.Append (cflags).Append (' ');
 				if (embed_mono) {
-					var libmono = "libmonosgen-2.0.a";
-					var lib = Path.Combine (libdir, libmono);
+					string libmono = Path.Combine (libdir, "libmonosgen-2.0.a");
 
-					if (!File.Exists (Path.Combine (lib)))
+					if (!File.Exists (Path.Combine (libmono)))
```

Comment:
not _new_ code - but there's no point in combining a single value ;-)
i.e. remove `Path.Combine`